ovoscope/__init__.py

```
@@ -114,39 +123,127 @@ def handle_end_of_test(msg: Message):
+                    r_dst = received.context.get("destination")
+                    assert e_src == r_src
+                    assert e_dst == r_dst
+                    if expected.msg_type in self.flip_points:
+                        e_src, e_dst = e_dst, e_src
+
+                if sess_e and sess_r:
+                    if self.test_session_lang:
+                        assert sess_e.lang == sess_r.lang
+                    if self.test_session_pipeline:
+                        assert sess_e.pipeline == sess_e.pipeline
+            except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)